includes/wikia/tests/UserPasswordTest.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+use Wikia\Service\Helios\ClientException;
+use Wikia\DependencyInjection\Injector;
+
+class UserPasswordTest extends WikiaBaseTest
+{
+
+	const TEST_USER_ID = 2257663;
+
+	/** @var User */
+	protected $testUser;
+
+	protected static $currentInjector;
+
+	public static function setUpBeforeClass() {
+		self::$currentInjector = Injector::getInjector();
+	}
+
+	public function setUp() {
+		parent::setUp();
+		$this->testUser = User::newFromId( self::TEST_USER_ID );
+	}
+
+	public static function tearDownAfterClass() {
+		Injector::setInjector( self::$currentInjector );
+	}
+
+	public function testShouldSetNewPassword() {
+		$this->assertTrue( $this->testUser->setPassword( "ok" ) );
+	}
+
+	/**
+	 * @expectedException PasswordError
+	 * @expectedExceptionMessage Passwords must be at least $1 characters.
+	 */
+	public function testShouldNotSetEmptyPassword() {
+		$this->assertTrue( $this->testUser->setPassword( "" ) );
+	}
+
+	/**
+	 * @expectedException PasswordError
+	 * @expectedExceptionMessage Your password must be different from your username.
+	 */
+	public function testShouldNotSetNewPasswordEqualToUsername() {
+		$this->assertTrue( $this->testUser->setPassword( $this->testUser->getName() ) );
+	}
+
+	public function testShouldNotSetNewPasswordForBlockedUser() {
+		$nameBkp = $this->testUser->getName();
+		$this->testUser->setName( "Useruser" );
```

Comment:
These tests can be removed per: https://github.com/Wikia/helios/pull/209#discussion_r86740213